presto-main/src/test/java/com/facebook/presto/type/TestArrayOperators.java

```diff
@@ -726,6 +735,15 @@ public void testArrayMax()
         assertDecimalFunction("ARRAY_MAX(ARRAY [2.22222222222222222, 2.3])", decimal("2.30000000000000000"));
     }
 
+    @Test
+    public void testArrayMaxWithNulls()
```

Comment:
These should be four separate test methods so they can pass or fail independently.